foundation.dd

```diff
@@ -39,29 +46,45 @@ $(UL
     $(LI $(HTTP acehreli.org, Ali Çehreli), Secretary)
 )
 
-$(H3 Becoming a member)
+$(H3 How can I contribute?)
 
-Individual and corporate membership packages will be announced in the future.
+$(P There are many ways to help the D programming language to grow further and
+    $(LINK2 https://wiki.dlang.org/Get_involved, get involved).)
 
-$(H3 Sponsoring D)
+$(P If you don't have the time to work on the source code,
+you can also $(LINK2 $(ROOT_DIR)donate.html, donate) to the D Language Foundation.)
 
-$(P If you want to help the D Language to advance further and have resources
-    that you can spend, don't hesitate to contact
-    $(HTTPS erdani.com/index.php/contact, Andrei) and
-    $(HTTP www.walterbright.com, Walter).
-    $(BR)
-    Your help is highly appreciated!
-)
+$(H3 The D Language Foundation is a non-profit organization)
 
-$(P We are pleased to announce that the D Language Foundation is now a 501(c)(3) tax exempt organization, and as such,
-    donations may be tax-deductible)
+The D Language Foundation has been recognized as a public charity under section 501(c)(3) of the Internal Revenue Code.
 
 $(H3 Official contacts)
 
 $(P The D Language Foundation has been incorporated with the state of Washington, USA.
     The employer ID is 47-5352856.)
 
+$(P If you want to get in touch with the D Language Foundation, please feel free to write to $(LINK2 mailto:foundation@dlang.org, foundation@dlang.org).)
+
+$(P For official correspondence, please write to:
+
+    $(BR)
+    $(DIVC dlang-address,
+        $(B The D Language Foundation)
+        $(BR)
+        13868 65th Place NE
+        $(BR)
+        Kirkland, WA 98034-4900
+        $(BR)
+    )
```

Comment:
Oh it seems that I didn't get the mail - I have rebased the PR, s.t. it's not included in the history.